app/packages/core/src/plugins/SchemaIO/components/TreeSelectionView.tsx

```diff
@@ -70,6 +71,13 @@ export default function TreeSelectionView(props: ViewPropsType) {
   const [checkedState, setCheckedState] =
     React.useState<CheckedState>(initialCheckedState);
 
+  const unboundState = useUnboundState(checkedState);
+
+  // useEffect(() => {
```

Comment:
If you are confident this useEffect is no longer needed, you can probably just delete it @lanzhenw - it looks like it is just a console.log() anyway

Reply:
removed.